benefits/static/css/styles.css

```diff
@@ -199,6 +201,13 @@ h2,
 h3,
 .h3 {
   font-size: var(--h3-font-size);
+  line-height: var(--heading-line-height);
+}
+
+/* H4 */
+/* Desktop only: Used for Agency Selector card, agency name */
+h4 {
+  line-height: var(--h4-line-height);
 }
```

Comment:
One could argue we should be declaring the H4 font sizes here, but I opted not to for this PR. I did add that comment, thought, that it's only used on Agency Selector card title. Next time we do use the H4, it can be refactored.

Reply:
I get that you're defining line-height on `h4` because the Agency Selector card title is the only place we actually use an `h4` element. And here you're saying that, by the same reasoning, we could also define all the styles for the Agency Selector card title on `h4`. 

I want to note that this is slightly different than what Figma says, since the Style Guide has `h4` with a certain font-size, line-height, etc. and the Components page has the Agency Selector Cards with a certain font-size, line-height, etc.

This is fine though since this code change gives us the net-effect we want and, as you noted,
> Next time we do use the H4, it can be refactored.

Reply:
Yea the next Letter-Spacing PR will apply the `h4` letter-spacing styles to this `h4`, and then the Agency Selector's H4 will get those styles. 

And we know we're gonna refactor the entire modal layout/includes after https://github.com/cal-itp/benefits/issues/1444 this ticket for SBMTD, we can revisit how the h4 on the modal is constructed and styled more thoroughly then.